subtrack.DAL/SubtrackDbContext.cs

```diff
@@ -18,5 +18,11 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             .HasValue<DateTimeSetting>(nameof(DateTimeSetting));
 
         modelBuilder.Entity<DateTimeSetting>().HasData(new DateTimeSetting { Id = DateTimeSetting.LastAutoPaymentTimeStampKey, Value = null });
+
+        modelBuilder.Entity<Subscription>(entity =>
```

Comment:
Dont need this as theese values will be default when creating a new subscription inside the Upsert page

Reply:
These will make sure any existing values have the expected default value. Unless it's a dev running this, I'd like to avoid deleting the values that I have added to the database.